oximeter/impl/benches/quantile.rs

```diff
@@ -0,0 +1,53 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use oximeter_impl::Quantile;
+use rand_distr::{Distribution, Normal};
+
+/// Emulates baseline code in a Python implementation of the P²
+/// algorithm:
+/// <https://github.com/rfrenoy/psquare/blob/master/tests/test_psquare.py#L47>.
+fn normal_distribution_percentile(size: i32, p: f64) -> f64 {
+    let mu = 500.;
+    let sigma = 100.;
+    let normal = Normal::new(mu, sigma);
+    let mut observations = (0..size)
+        .map(|_| normal.unwrap().sample(&mut rand::thread_rng()))
+        .collect::<Vec<f64>>();
+    float_ord::sort(&mut observations);
+    let idx = ((f64::from(size) - 1.) * p) as usize;
+    observations[idx]
+}
+
+fn normal_distribution_quantile(size: i32, p: f64) -> f64 {
+    let mu = 500.;
+    let sigma = 100.;
+    let normal = Normal::new(mu, sigma);
+    let mut observations = (0..size)
+        .map(|_| normal.unwrap().sample(&mut rand::thread_rng()))
```

Comment:
This calls `unwrap()` inside the loop, which copies `normal` on each iteration. We should call `unwrap()` just on L23.